src/Model/PhasorDynamics/SynchronousMachine/GENROUwS/GenrouData.hpp

```diff
@@ -39,6 +39,8 @@ namespace GridKit
     enum class GenrouPorts
     {
       bus,             ///< Unique ID of the connecting bus
+      pmech_signal,    ///< Unique ID of the governor Pmech signal.
+      speed_signal,    ///< Unique ID of the speed signal of the machine.
```

Comment:
we should also modify `src/Model/PhasorDynamics/INPUT_FORMAT.md` if we're adding new ports